### PR TITLE
chore(docs): Documentation update of Azure and debugging in VSCode

### PR DESCRIPTION
**Pull request by pedrooot**
### Context

This pr adds info about azure set-up and debugging in VSCode 

### Description

Readme files are modified

Feat @puchy22 

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/developer-guide/debugging.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Debugging
+
+Debugging in prowler make things easier!
+Here is an example of the debugging configuration file that you can add to vscode to debug prowler:
```
Comment (jfagoagas) [inline docs/developer-guide/debugging.md:4]: - Where should I put that file to let VSCode know that this is my configuration?
- What should be the name of that file?

Comment (jfagoagas) [inline docs/developer-guide/debugging.md:4]:
```suggestion
If you are developing Prowler, it's possible that you will encounter some situations where you have to inspect the code in depth to fix some unexpected issues during the execution. To do that, if you are using VSCode you can run the code using the integrated debugger. Please, refer to this [documentation](https://code.visualstudio.com/docs/editor/debugging) for guidance about the debugger in VSCode.

The following file is an example of the [debugging configuration](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations) file that you can add to [Virtual Studio Code](https://code.visualstudio.com/):
```

File: docs/developer-guide/debugging.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Debugging
+
+Debugging in prowler make things easier!
```
Comment (jfagoagas) [inline docs/developer-guide/debugging.md:3]:
```suggestion
Debugging in Prowler make things easier!
```

File: docs/developer-guide/debugging.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Here is an example of the debugging configuration file that you can add to vscode to debug prowler:
+
+```json
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (jfagoagas) [inline docs/developer-guide/debugging.md:10]:
```suggestion
```

File: docs/getting-started/requirements.md [hunk 4]
```diff
@@ -71,14 +71,20 @@ To use each one you need to pass the proper flag to the execution. Prowler fro A
 
 #### Azure Active Directory scope
 
-Azure Active Directory (AAD) permissions required by the tool are the following:
+Microsoft Entra ID (AAD earlier) permissions required by the tool are the following:
 
 - `Directory.Read.All`
 - `Policy.Read.All`
 
 The best way to assign it is through the azure web console:
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md:79]:
```suggestion
The best way to assign it is through the azure web console:

```

Comment (jfagoagas) [reply docs/getting-started/requirements.md:79]: You have to include a new line for the numbered list to be formatted.

File: docs/getting-started/requirements.md [hunk 5]
```diff
@@ -71,14 +71,20 @@ To use each one you need to pass the proper flag to the execution. Prowler fro A
...
+1. Access to Microsoft Entra ID > App registrations and register an application
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md:80]:
```suggestion
1. Access to Microsoft Entra ID, then go to "App registrations" and register a new application
```

File: docs/getting-started/requirements.md [hunk 6]
```diff
@@ -71,14 +71,20 @@ To use each one you need to pass the proper flag to the execution. Prowler fro A
...
+  ![Register an Application page](../img/register-application.png)
+2. Once our application is registered, select *Api permissions* blade and add the needed permissions:
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md:82]:
```suggestion
2. Once your application is registered, select "API permissions" blade and add the needed permissions:
```

Comment (jfagoagas) [reply docs/getting-started/requirements.md:82]: What do you mean by "blade"?

File: docs/getting-started/requirements.md [hunk 7]
```diff
@@ -87,6 +93,18 @@ Regarding the subscription scope, Prowler by default scans all the subscriptions
 - `Security Reader`
 - `Reader`
 
+To assign this roles, follow the instructions:
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md:96]:
```suggestion
To assign this roles, follow the instructions:

```

File: docs/getting-started/requirements.md [hunk 8]
```diff
@@ -87,6 +93,18 @@ Regarding the subscription scope, Prowler by default scans all the subscriptions
...
+1. Access Subscription > Select your subscription > Access control (IAM) > Roles
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md:97]: Please detail more these steps.

File: docs/getting-started/requirements.md [hunk 9]
```diff
@@ -87,6 +93,18 @@ Regarding the subscription scope, Prowler by default scans all the subscriptions
...
+  ![IAM Page](../img/page-IAM.png)
+2. Click on Add and select 'Add role assigment'
+3. In the bar, search `Security Reader` and select it.
+4. In the Members blade, select a member.
+5. Review + assign
+
+*Repeat these steps for `Reader`role*
+
+
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md:106]:
```suggestion
```

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: docs/developer-guide/debugging.md (hunk 1 above)
Comment (pedrooot) [reply docs/developer-guide/debugging.md:4]: Done

Comment (pedrooot) [reply docs/developer-guide/debugging.md:4]: Done

File: docs/developer-guide/debugging.md (hunk 2 above)
Comment (pedrooot) [reply docs/developer-guide/debugging.md:3]: Done

File: docs/getting-started/requirements.md (hunk 4 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md:79]: Done

File: docs/getting-started/requirements.md (hunk 5 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md:80]: Done

File: docs/getting-started/requirements.md (hunk 6 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md:82]: Done, a blade is the item of the page that shows a new page

File: docs/getting-started/requirements.md (hunk 7 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md:96]: Done

File: docs/getting-started/requirements.md (hunk 8 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md:97]: Done

File: docs/getting-started/requirements.md (hunk 9 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md:106]: Done

**Comment by jfagoagas** [comment]
Closing this in favor of https://github.com/prowler-cloud/prowler/pull/3411

**Pull request closed without merging**